ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/ConnectIntegrationTest.java

```diff
@@ -223,6 +226,40 @@ public void shouldDropConnector() {
         is("mock-connector"));
   }
 
+  @Test
+  public void shouldCreateSourceConnectorSuccessfully() {
+    // Given:
+    String connectorName = "source-mock-connector";
+    connectNames.add(connectorName);
```

Comment:
Ah interesting! I didn't realize that the `create()` helper method in this integration test already tested the CREATE CONNECTOR functionality. It'd be good to re-use that helper method in this test, rather than repeating the code. You can modify the helper method to return the `RestResponse` in order to perform the additional checks in these tests that you've added.